rules/aws/initial_access_root_console_failure_brute_force.toml

```diff
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2020/07/21"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/07/21"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a high number of failed authentication attempts to the AWS management console for the Root user identity. An
+adversary may attempt to brute force the password for the Root user identity, as it has complete access to all services
+and resources for the AWS account.
+"""
+from = "now-20m"
+index = ["filebeat-*"]
+language = "kuery"
+license = "Elastic License"
+name = "AWS Management Console Brute Force of Root User Identity"
+note = "The AWS Filebeat module must be enabled to use this rule."
+references = ["https://docs.aws.amazon.com/IAM/latest/UserGuide/id_root-user.html"]
+risk_score = 73
+rule_id = "4d50a94f-2844-43fa-8395-6afbd5e1c5ef"
+severity = "high"
+tags = ["AWS", "Continuous Monitoring", "Elastic", "Identity and Access", "SecOps"]
+type = "threshold"
+
+query = '''
+event.module:aws and event.dataset:aws.cloudtrail and event.provider:signin.amazonaws.com and event.action:ConsoleLogin and aws.cloudtrail.user_identity.type:Root and event.outcome:failure
+'''
+
+
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+[[rule.threat.technique]]
+id = "T1110"
+name = "Brute Force"
+reference = "https://attack.mitre.org/techniques/T1110/"
+
+
+[rule.threat.tactic]
+id = "TA0006"
+name = "Credential Access"
+reference = "https://attack.mitre.org/tactics/TA0006/"
+
+[rule.threshold]
+field = ""
```

Comment:
is there an environment ID we can use to bind this to in case multiple disparate accounts/environments are ingested

Reply:
there should always be an account ID in the `cloud.account.id` field but IDT we can require that value be the same across the events matching the threshold count?

Reply:
if we dont use account id, i think the only other fields that may make sense would be something to do with the host that made the failed requests - `host.name`, `host.id`, `host.mac`, `host.ip`. thoughts on account id versus host info?